RecoParticleFlow/PFProducer/plugins/PFCandidateChecker.cc

```diff
@@ -1,13 +1,86 @@
-#include "RecoParticleFlow/PFProducer/plugins/PFCandidateChecker.h"
+/**\class PFCandidateChecker 
+\brief Checks what a re-reco changes in PFCandidates.
+
+\author Patrick Janot
+\date   August 2011
+*/
+
 #include "DataFormats/ParticleFlowCandidate/interface/PFCandidate.h"
 #include "DataFormats/ParticleFlowReco/interface/PFBlock.h"
 #include "DataFormats/JetReco/interface/PFJet.h"
-
 #include "FWCore/Framework/interface/ESHandle.h"
-
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 #include "FWCore/Utilities/interface/Exception.h"
 #include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDAnalyzer.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "DataFormats/ParticleFlowCandidate/interface/PFCandidate.h"
```

Comment:
Remove duplicate inclusion (and perhaps also arrange by putting DataFormat and FWCore headers together